cypress/integration/metaPersianSpec.js

```diff
@@ -17,6 +18,12 @@ describe('Persian Article Meta Tests', () => {
     metaElement.should('have.attr', 'content', 'nofollow');
   });
 
+  it('should have description meta data', () => {
+    metaDataDescription(
```

Comment:
@jamesbrumpton Instead of creating a new testHelper function, you could use your existing function 
```
retrieveMetaDataContent(
    'head meta[name="description"]', 
    'شاید خیلی طول نکشد که زمانی برسد که وقتی خسته هستید و مثلا هوس فنجان قهوه‌ای را کردید، پهپادی را ببینید که با قهوه سراغتان می‌آید.'
);
```

Reply:
We discussed this offline. James said having a test helper named `metaDataDescription` is more useful for the Test team, so I am happy to keep this in the codebase for now.